### ISSUE TITLE
Virginia - Receiver Onboarding

### ISSUE THREAD

**Issue by LaFreda1** [CDCgov/prime-reportstream#11476]
### What You Need to Know 

- [ ] Terms of Service agreement signed 
- [ ] Kickoff meeting completed 
- [ ] Receiver intake form completed: [paste info below this] 

### To Do 

- [ ] create individual tickets for this epic 
- [ ] RS team member who originates this ticket: add any dependencies, as needed
- [ ] RS team member who originates this ticket: complete the following when creating this Issue. 
     - [ ] Pipelines: Onboarding & Operations (New) 
     - [ ] Labels: "Onboarding-ops" and "Receiver" 
     - [ ] Epics: [select the epic for this receiver we're onboarding]

**Comment by sliu1000** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcWpLIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--40a4117038fd28612eb8b0e61c7877d7613caa01/2023-09-22%20VA.png)

**Comment by sliu1000** [comment]
@LaFreda1 did we decide which engineer will be working on this?

**Comment by LaFreda1** [comment]
No, Virginia will be onboarding to UP, and not COVID. Once I know the game plan for UP I will assign someone.
